subsys/net/lib/nrf_cloud/src/nrf_cloud_fsm.c

```diff
@@ -258,7 +258,11 @@ static int handle_device_control_update(const struct nct_evt *const evt,
 		return -ENOENT;
 	}
 
+#if IS_ENABLED(CONFIG_NRF_CLOUD_ALERTS)
```

Comment:
Could `nrf_cloud_alert_control_get` be protected in the same way as `nrf_cloud_rest_alert_send`?

Then you could either remove these guards, or if the intention is to keep them for clarity, replace them with `IS_ENABLED`

Reply:
Actually I'd probably protect nrf_cloud_alert_control_set too just for good measure, so that the entire module can be used in this manner

Reply:
I will consider that in a follow-on PR.  This needs to get merged to unblock CI.